MiniAn/minian_run.py

```diff
@@ -437,22 +438,23 @@
         c0 = save_minian(c0_new.rename("c0").chunk({"unit_id": 1, "frame": -1}), intpath, overwrite=True)
         A = A.sel(unit_id=C.coords["unit_id"].values)
         # 5. Merge components
-        print(RUN_CNMF_MERG_COMP.fromat("1st"), flush=True)
+        print(RUN_CNMF_MERG_COMP.format("1st"), flush=True)
         try:
             A_mrg, C_mrg, [sig_mrg] = unit_merge(A, C, [C + b0 + c0], **params_unit_merge)
         except TypeError:
-            print_to_intercept(ONE_PARM_WRONG_TYPE.format("unit_merge"))
+            utils.print_to_intercept(ONE_PARM_WRONG_TYPE.format("unit_merge"))
             sys.exit()
         # Save
-        print(RUN_CNMF_SAVE_INTERMED.fromat("1st"), flush=True)
+        print(RUN_CNMF_SAVE_INTERMED.format("1st"), flush=True)
         A = save_minian(A_mrg.rename("A_mrg"), intpath, overwrite=True)
         C = save_minian(C_mrg.rename("C_mrg"), intpath, overwrite=True)
         C_chk = save_minian(C.rename("C_mrg_chk"), intpath, overwrite=True,
                             chunks={"unit_id": -1, "frame": chk["frame"]})
         sig = save_minian(sig_mrg.rename("sig_mrg"), intpath, overwrite=True)
 
-    except:
-        print_to_intercept(NO_CELLS_FOUND)
+    except Exception as error:
+        utils.print_error(error, RUN_CNMF_ITT.format("1st"))
```

Comment:
If I understand right the text will be "Error in Running CNMF 1st itteration::  " -- double "::" and double "  ". I think it doesn't hurt to add one more definition to make the text more readable.
Also, there is one t in the word iteration.

Reply:
I could also edit the text and do something like that?
```
RUN_CNMF_ITT                = "Running CNMF {0} itteration"
RUN_CNMF_ESTIM_NOISE        = RUN_CNMF_ITT + ": estimating noise..."
```

I did something similar at one point

Reply:
I would just write `"CNMF {0} iteration"`